worker/lease/dead_manager_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package lease_test
+
+import (
+	"reflect"
+
+	gc "gopkg.in/check.v1"
+
+	lease "github.com/juju/juju/worker/lease"
+)
+
+type DeadManagerSuite struct {
+}
+
+var _ = gc.Suite(&DeadManagerSuite{})
+
+type DeadManagerError struct{}
+
+const DeadManagerErrorMessage = "lease manager stopped"
+
+func (*DeadManagerError) Error() string {
+	return DeadManagerErrorMessage
+}
+
+// This creates and new DeadManager and calls all of its exported methods with zero
+// values. All methods should return the specified DeadManagerError.
+func (s *DeadManagerSuite) TestDeadManager(c *gc.C) {
```

Comment:
I think that the generality of this method is both needed and necessary - a new exported method added will also be checked.